client/stylesheets/_responsive.scss

```diff
@@ -263,6 +372,11 @@
 }
 
 @media (max-width: 500px) {
+    .searchResultsCardImg {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `searchResultsCardImg` should be written in lowercase with hyphens